.editorconfig

```diff
@@ -11,3 +11,7 @@ insert_final_newline = true
 [*.hs]
 indent_size = 2
 max_line_length = 80
+
+[*.py]
+indent_size = 4
```

Comment:
Why would we want to use a different indentation than in haskell?

Reply:
And where are these promised `.py` files? :D

Reply:
Ah! I started writing some python script for the validation until I realize there was actually a CLI already available for the target library :upside_down_face:. And, to answer the first question, 4 spaces is quite common in Python where indentation is all that matters for defining structure of the code. It ain't like Haskell poetry where every single Haskeller has its own habits and customs as for how to indent things. 

For reference, here's the validation script (could be useful one day, who knows :shrug:)

```py
#!/usr/bin/python

import sys
import json
from urllib.request import urlopen
from getopt import getopt, GetoptError
import jsonschema

def main(inputFile):
    with open(inputFile, 'r') as file:
        data=json.loads(file.read())
    schemaUrl = data["$schema"]
    schema = json.loads(urlopen(schemaUrl).read().decode("utf-8"))
    jsonschema.validate(data, schema)

def parseOptions (argv):
    usage = "Usage:\n\tvalidate-json.py -i <input-file>"

    try:
        opts, args = getopt(argv,"hi:",["input-file="])
    except GetoptError:
        print("Invalid option.")
        print(usage)
        sys.exit(2)

    for opt, arg in opts:
        if opt == '-h':
            print(usage)
            sys.exit()
        elif opt in ("-i", "--ifile"):
            return arg

    print("Missing option.")
    print(usage)
    sys.exit(2)

if __name__ == "__main__":
    main(parseOptions(sys.argv[1:]))
```